src/model/RacingCar.js

```diff
@@ -0,0 +1,28 @@
+const { SIGN } = require('../utils/constants');
+const { canMove } = require('../utils/movementGenerator');
+
+class RacingCar {
+  #name;
+  #moveResult = [];
+
+  constructor(name) {
+    this.#name = name;
+    this.#moveResult = [];
+  }
+
+  getName() {
+    return this.#name;
+  }
+
+  getMoveResult() {
+    return this.#moveResult.join('');
+  }
+
+  move() {
+    if (canMove()) {
+      this.#moveResult.push(SIGN.movement);
```

Comment:
```suggestion
      this.#moveResult += SIGN.movement;
```
moveResult를 배열로 관리해야할 필요가 있나? 실행 안시켜봐서 잘 모르겠지만 굳이 그럴필요 없으면 스트링으로 관리하는게 낫지. push가 성능이 안좋기도 하고.